src/App.js

```diff
@@ -15,14 +23,18 @@ class App extends Component {
   render() {
     return (
       <div>
+        <MuiThemeProvider>
         <header>
-          <headerNav></headerNav>
-          <Link to="/">Home</Link>
+
         </header>
 
         <main>
-          <Route exact path="/" component={home} />
-        </main>
+            <Route exact path="/" component={home} />
+            <Route exact path="/nav" component={headerNav} />
```

Comment:
headerNav should be HeaderNav -> remember capitalization

Reply:
Removed it. was conflicting with other stuff